atum/src/main/scala/za/co/absa/atum/AtumImplicits.scala

```diff
@@ -269,6 +269,16 @@ trait AtumImplicitsBase {
       }
     }
 
+    /**
+     * The method sets an additional info in metadata of the _INFO file with key - value pair
+     */
```

Comment:
Sorry I didn't tell you the trick. In IntelliJ, when you write `/**` above the function definition or class declaration the IDE will pre-fill the comments for you. Line (or more for the function description, the input parameters and eventually the return value to add their description.

```suggestion
  /**
    * Sets an additional info in metadata of the _INFO file
    * @param keyAndValue - the pair of the _key_ and _value_ to add to the _INFO file, in a form of tuple
    * @param replaceIfExists - flag to indicate if the value should be replaced in the case the key already exists
    * @return - the original `DataFrame` the method was called upon
    */     
    
    This is the standard form for in-code documentation, called [ScalaDoc](https://docs.scala-lang.org/style/scaladoc.html). There are tools that can extract it to outside documentation in case of need.
```

Reply:
Updated, thanks for the tip.